src/main/java/com/piekoszek/nowaksharedrent/config/ExceptionHandlerController.java

```diff
@@ -0,0 +1,24 @@
+package com.piekoszek.nowaksharedrent.config;
+
+import com.piekoszek.nowaksharedrent.invite.exceptions.InviteCreatorException;
+import com.piekoszek.nowaksharedrent.jwt.exceptions.InvalidTokenException;
+import com.piekoszek.nowaksharedrent.response.MessageResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+@ControllerAdvice
+class ExceptionHandlerController {
```

Comment:
Add some method for Throwable.class, and send InternalServerError response with some kind message, like "Error, we are working on that" :D